distribution/docs/src/main/resources/content/_managing/attributes-added-by-input-transformers.adoc

```diff
@@ -0,0 +1,27 @@
+:title: Attributes Added by Input Transformers
+:type: dataManagement
+:status: published
+:parent: Automatically Added Metacard Attributes
+:summary: How input tranformers add attributes to metacards.
+:order: 01
+
+=== {title}
+
+<<_available_input_transformers,Input transformers>> create and populate metacards from a resource.
+
+{branding} chooses which input transformer to use by:
+
+. Resolving the mimetype for the resource. This is done by trying the following steps in order until a proper mimetype is found:
+  .. Extracting the optionally given mimetype from the resource.
+  .. Passing the resource's file extension to a <<_mime_type_mapper,mimetype mapper>>.
+  .. Using https://tika.apache.org/[Apache Tika].
+  .. Manually parsing the resource's file contents for the first character. For example: if the file starts with `<` it is assumed to be XML.
+. Gathering all of the input transformers associated with the resolved mimetype.
+. Iterating through the transformers until a successful transformation is performed.
+
+The first transformer that can successfully create a metacard from the ingested resource is chosen. If no transformer can successfully transform the resource, the ingestion process fails.
+
+There are two exceptions to this strategy:
+
+* The <<_configuring_the_content_directory_monitor,Content Directory Monitor>> only performs the first two steps of resolving the mime type.
+* The <<_ingest_command,ingest command>> will default to the XML input transformer instead of failing.
```

Comment:
❗️ It should be mentioned that the RestEndpoint that backs Intrigue will skip all of these steps and use the mimetype sent over HTTP, if it's present.

Reply:
Very good catch!